app/views/claim_mailer/approved.text.erb

```diff
@@ -0,0 +1,30 @@
+Dear <%= @display_name %>,
+
+We are pleased to say that your claim to get your student loan payments has been approved.
+
+We will now process your claim so we can calculate the amount of tax owed for this repayment, and then pay it for you. This will not affect the amount of money you will receive.
+
+To do this we will share your claim data with our payroll provider and HMRC. This will include:
+
+* name
+* email
+* address
+* date of birth
+* gender as recorded on your digital identity
+* student loan plan
+* teacher reference number
+* bank details
+
+Some of this information was provided by you when you made your claim, the rest of the information was obtained from your records held within the Department for Education.
+
+# Next steps
+
+We will contact you again when we are ready to send the payment.
+
+# Contact us if something is incorrect
```

Comment:
How would I know? We're not playing the data back to the user.

Reply:
Would it be safe to play back all that PII in an email? I thought that as long as they knew what kind of information was being sent that would that would be enough? 🤷‍♂

Reply:
Unless I've got a screenshot of the check your answers page, I don't have the information I've submitted, so I don't know what I'm correcting. There's not even a date so I can think about what might have changed since I submitted my claim.

Maybe the solution for now is:
```suggestion
# Contact us if something has changed
```